src/main/java/com/flexrate/flexrate_back/member/dto/SignupRequestDTO.java

```diff
@@ -0,0 +1,46 @@
+package com.flexrate.flexrate_back.member.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.flexrate.flexrate_back.member.enums.ConsumptionType;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.time.LocalDate;
+
+/*
+ * 회원가입 요청 시 사용되는 DTO
+ * @since 2025.04.28
+ * @author 윤영찬
+ */
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class SignupRequestDTO {
+
+    @Email(message = "유효한 이메일을 입력해 주세요.")
+    @NotBlank(message = "이메일은 필수 항목입니다.")
+    private String email;
+
+    @NotBlank(message = "비밀번호는 필수 항목입니다.")
+    private String password;
+
+    @NotBlank(message = "성별은 필수 항목입니다.")
+    private String sex;
+
+    @NotBlank(message = "이름은 필수 항목입니다.")
+    private String name;
+
+    @NotNull(message = "생년월일은 필수 항목입니다.")
+    @JsonFormat(pattern = "yyyy-MM-dd")
+    private LocalDate birthDate;
+
+    @NotNull(message = "소비성향은 필수 항목입니다.")
+    private ConsumptionType consumptionType;
+
+    @NotBlank(message = "소비 목표는 필수 항목입니다.")
+    private String consumptionGoal;
+
+}
```

Comment:
기존 코드들을 참고하시면, DTO들에는 record를 사용하고있어요!
record는 불변 객체로, 생성 후 내부 상태를 변경할 수 없어 데이터의 일관성을 보장해주고, 다중 스레드 환경에서 안전하게 사용할 수 있도록 해줘용.
또한 getter도 자동으로 생성해주니 @Getter를 선언하지 않아도 돼요.
컨벤션을 맞출수있도록 변경해주세요!

```suggestion
@Builder
public record SignupRequestDTO (
        @Email(message = "유효한 이메일을 입력해 주세요.")
        @NotBlank(message = "이메일은 필수 항목입니다.")
        String email,

        @NotBlank(message = "비밀번호는 필수 항목입니다.")
        String password,

        @NotBlank(message = "성별은 필수 항목입니다.")
        String sex,

        @NotBlank(message = "이름은 필수 항목입니다.")
        String name,

        @NotNull(message = "생년월일은 필수 항목입니다.")
        @JsonFormat(pattern = "yyyy-MM-dd")
        LocalDate birthDate,

        @NotNull(message = "소비성향은 필수 항목입니다.")
        ConsumptionType consumptionType,

        @NotBlank(message = "소비 목표는 필수 항목입니다.")
        String consumptionGoal
) {}
```